src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    white-space: nowrap;
+    color: $colorMainText;
+  }
+
+  &__link:visited {
+    color: $colorMainText;
+  }
+
+  &__link:hover {
+    color: $colorMain;
```

Comment:
Ensure that the variable `$colorMain` is defined in your SCSS files. Otherwise, this will cause a compilation error.